components/navbar/navbar.css

```diff
@@ -55,8 +55,7 @@
 }
 
 .ampstart-navbar-trigger {
-  line-height: var(--navbar-height);
-  font-size: var(--h2);
+  height: 24px;
```

Comment:
Note: Try to replace with @equiv; and see if Samsung device renders correctly. http://graphemica.com/%E2%89%A1